qubes/vm/qubesvm.py

```diff
@@ -442,6 +442,18 @@ class QubesVM(qubes.vm.mix.net.NetVMMixin, qubes.vm.LocalVM):
             :param subject: Event emitter (the qube object)
             :param event: Event name (``'domain-unpaused'``)
 
+        .. event:: domain-post-unpaused (subject, event)
+
+            Fired when the domain has been unpaused. The event
+            ``domain-unpaused`` already signals that the unpause has completed,
+            but the caller can't fire async events and wait for its results,
+            therefore this event exists.
+
+            Handler for this event may be asynchronous.
+
+            :param subject: Event emitter (the qube object)
+            :param event: Event name (``'domain-post-unpaused'``)
+
```

Comment:
The problem with this approach (called from start/unpause functions) is it won't really synchronize with libvirt events, which has two problems:
- if vm is unpaused outside of qubesd for some reason (virsh unpause?) - the event won't be called at all
- the `domain-unpaused` may arrive after `domain-post-unpaused` due to how events are processed

If the async event is needed only for the script I requested moving to tests only, then the event is not needed anymore. But if it's still necessary, this will need a better solution, probably re-arrange code to make domain-unpaused async already.